src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle() {
+        super();
+        setName(FigureType.CIRCLE);
+        this.radius = 1;
+    }
+
+    public Circle(FigureType name, String color, double radius) {
+        super(name, color);

```

Comment:
We don't need 'figure name' argument here. It is always the same here
```suggestion
    public Circle(String color, double radius) {
        super(FigureType.CIRCLE, color);
```